modules/api/src/main/java/org/eclipse/xpanse/api/controllers/TerraformBootWebhookApi.java

```diff
@@ -45,14 +45,16 @@ public class TerraformBootWebhookApi {
     @Tag(name = "Webhook", description = "Webhook APIs")
     @Operation(description = "Process the execution result after terraform executes the command "
             + "line.")
-    @PostMapping(value = "${webhook.deployCallbackUri}/{task_id}", produces =
+    @PostMapping(value = "${webhook.deployCallbackUri}{task_id}", produces =
             MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.OK)
     public void deployCallback(
             @Parameter(name = "task_id", description = "task id")
-            @PathVariable("task_id") String taskId, @Valid @RequestBody TerraformResult result) {
+            @PathVariable("task_id") String taskId,
+            @Valid @RequestBody TerraformResult result) {
 
-        terraformDeploymentResultCallbackManager.deployCallback(taskId, result);
+        terraformDeploymentResultCallbackManager.deployCallback(taskId, result,
+                result.getProcessInstanceId());
```

Comment:
there should be no changes to terraform-boot. The "processInstanceId" is internal to xpanse and terraformBoot has no idea about it. 

We must just check if  the taskId has any related old service ID in the migration table. If yes, then we additionally push the workflow for the corresponding migration ID. 

same for destroy.